### PR TITLE
Add camel-azure-eventhubs wrapper

### PR DESCRIPTION
**Pull request by lopushen**

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -482,6 +482,20 @@
 <!--        <bundle dependency='true'>wrap:mvn:com.amazonaws/aws-xray-recorder-sdk-sql-mysql/${aws-xray-version}</bundle>-->
 <!--        <bundle>mvn:org.apache.camel/camel-aws-xray/${project.version}</bundle>-->
 <!--    </feature>-->
+    <feature name='camel-azure-eventhubs' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <bundle dependency='true'>mvn:org.apache.camel.karaf/camel-cloudevents/${project.version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-core-amqp/2.9.1</bundle>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-core/1.46.0</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml:505]: `<bundle dependency='true'>wrap:mvn:com.azure/azure-core/1.46.0</bundle> `
and
`<bundle dependency='true'>wrap:mvn:com.azure/azure-storage-common/12.24.2</bundle>`
are both used in 4 places, same versions - sometimes an older version is used (azure-core 1.45.1 in camel-azure-storage-queue, but dependency tree now suggests version 1.46.0 - maybe has been updated in karaf)

@essobedo should we make it a feature?

Comment (essobedo) [reply features/src/main/feature/camel-features.xml:505]: 👍 Yes good idea

Comment (essobedo) [reply features/src/main/feature/camel-features.xml:505]: Let's do it in a different PR once all azure features will be merged

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**